### PR TITLE
fix: correct setting order of columns when we have more than 10 columns

### PR DESCRIPTION
**Pull request by hdamr**
Fixes https://github.com/vaadin/web-components/issues/1343

### CODE REVIEW

File: test/column-groups.html [hunk 1]
```diff
@@ -493,9 +860,45 @@
         });
 
       });
+
+      describe('Large nested groups', () => {
+        let grid;
+
+        beforeEach(() => {
+          const items = Array.from(Array(13).keys()).slice(1);
```
Comment (web-padawan) [inline test/column-groups.html:868]: This line can be removed.

File: test/column-groups.html [hunk 2]
```diff
@@ -493,9 +860,45 @@
...
+          expect(col19._order).to.be.lessThan(col20._order);
+        });
+      });
+
+      describe('More than 100 columns', () => {
+        let grid;
+        let items;
+
+        beforeEach(() => {
+          items = Array.from(Array(101).keys());
```
Comment (web-padawan) [inline test/column-groups.html:887]: This line can be removed.

File: test/column-groups.html [hunk 3]
```diff
@@ -493,9 +860,45 @@
...
+          const col19 = grid.querySelector('#col19');
+          const col20 = grid.querySelector('#col20');
+
+          expect(col19._order).to.be.lessThan(col20._order);
+        });
+      });
+
+      describe('More than 100 columns', () => {
+        let grid;
+        let items;
```
Comment (web-padawan) [inline test/column-groups.html:884]: This line can be removed.

**Comment by sosa-vaadin** [comment]
This PR also fixes https://github.com/vaadin/flow-components/issues/1017

**Review by sosa-vaadin — APPROVED** [review]

**Pull request merged**